mkosi

```diff
@@ -2239,7 +2241,10 @@ def run_postinst_script(args: CommandLineArguments, workspace: str, do_run_build
         shutil.copy2(args.postinst_script,
                      os.path.join(workspace, "root", "root/postinst"))
 
-        run_workspace_command(args, workspace, "/root/postinst", verb, network=args.with_network)
+        env = {
+            "CARGO_HOME": "/var/cache/cargo",
+        }
```

Comment:
Why not simply mount the cache to root/.cargo? Then we don't need all the CARGO_HOME stuff.